release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1636,6 +1636,42 @@ With this release, availability sets are no longer modified after creation, allo
+
+* Previously, the shared Timestamp component referenced an undefined property when calculating relative times. As a result, most times displayed in the console were not correctly displaying relative strings such as "Just now" or "Less than a minute ago". With this release, the issue is fixed and relative time strings are correctly rendered in the console. (link:https://issues.redhat.com/browse/OCPBUGS-51202[OCPBUGS-51202])
+
+* Previously, the observe menu only displayed based on the current user and console configuration for monitoring. This caused other items added by observability plugins to be hidden. With this release, the observe menu also displays items from different observability plugins. (link:https://issues.redhat.com/browse/OCPBUGS-50693[OCPBUGS-50693])
+
+* Previously, when logging in to the console for the first time, automatic perspective detection caused the console to ignore the specific URL path that the user clicked to get onto the console, and instead load a different page. With this release, the current path is followed even when automatic perspective switching occurs. (link:https://issues.redhat.com/browse/OCPBUGS-50650[OCPBUGS-50650])
+
+* Previously, there was an issue when creating new tabs in the horizontal navigation present in the web console from plugins. With this release, you can use plugins to create tabs on the web console horizontal navigation. (link:https://issues.redhat.com/browse/OCPBUGS-49996[OCPBUGS-49996])
+
+* Previously, the *Cluster Settings* page would not properly render during a cluster update if the `ClusterVersion` did not receive a `Completed` update. With this release, the *Cluster Setting* page properly renders even if the `ClusterVersion` has not received a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-49839[OCPBUGS-49839]) 
+
+* Previously, the links on the CLI downloads page were not sorted by operating system. With this release, the links are sorted by their operating system in alphabetical order. (link:https://issues.redhat.com/browse/OCPBUGS-48413[OCPBUGS-48413])
+
+* Previously, multiple external link icons could appear in the primary *Action* button of the *OperatorHub* modal. With this release, only a single external link icon appears. (link:https://issues.redhat.com/browse/OCPBUGS-46555[OCPBUGS-46555]) 
+
+* Previously, clicking the `Don't show again` link in the Lightspeed modal did not correctly navigate to the general *User Preference* tab when one of the other *User Preference* tabs was displayed. After this update, clicking the `Don't show again` link correctly navigates to the general *User Preference* tab. (link:https://issues.redhat.com/browse/OCPBUGS-46511[(*OCPBUGS-46511)] 
```

Comment:
oh and there is an attribute for lightspeed too.
ols: OpenShift Lightspeed
ols-official: Red{nbsp}Hat OpenShift Lightspeed